sandbox/rocky/tf/optimizers/first_order_optimizer.py

```diff
@@ -1,17 +1,15 @@
+import time
 
+import pyprind
+import tensorflow as tf
 
-
+from rllab.core import Serializable
 from rllab.misc import ext
 from rllab.misc import logger
-from rllab.core import Serializable
+from rllab.optimizers import BatchDataset
 from sandbox.rocky.tf.misc import tensor_utils
```

Comment:
It would be better to insert an empty line into rllab and sandbox to separate them into two different groups.